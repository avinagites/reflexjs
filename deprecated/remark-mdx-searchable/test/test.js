const fs = require("fs")
const path = require("path")
const remark = require("remark")
const mdx = require("remark-mdx")
const searchable = require("..")

const doc = fs.readFileSync(path.join(__dirname, "./fixture.mdx"), "utf8")

it("extracts searchable sections from mdx", () => {
  const result = remark().use(mdx).use(searchable).processSync(doc)

  expect(result.data).toMatchSnapshot()
})
