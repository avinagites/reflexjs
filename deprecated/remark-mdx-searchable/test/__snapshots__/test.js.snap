// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`extracts searchable sections from mdx 1`] = `
Array [
  Object {
    "heading": null,
    "text": "This is a paragraph with no section",
  },
  Object {
    "heading": "Section heading 1",
    "text": "This is paragraph 1 with emphasied code.",
  },
  Object {
    "heading": "Section heading 1",
    "text": "This is paragraph 2 with inline code.",
  },
  Object {
    "heading": "Section heading 2",
    "text": "This is content from section heading 2 with a link and some bold text.",
  },
  Object {
    "heading": "Section Heading 3",
    "text": "This is some content for section 3.",
  },
]
`;
